src/deluge/deluge.cpp

```diff
@@ -477,6 +479,10 @@ void setupStartupSong() {
 		else {
 			setupBlankSong();
 		}
+	} break;
+	case StartupSongMode::BLANK:
```

Comment:
```suggestion
	case StartupSongMode::BLANK:
	    [[fallthrough]];
```

Reply:
interestingly the compiler does not yell at me even tho i dont set this